README.md

```diff
@@ -21,6 +21,12 @@ Inside the project, install the project's dependencies:
 cd igbo_api/
 yarn install
 ```
+If you are running the API on Windows operating system, run the module to install it globally:
+
+```
+cd igbo_api/                   
+npm install --save-optional win-node-env
```

Comment:
We don't want to locally save this package into the project. You were right the first time, the command that should be used is:

`npm install -g win-node-env`

The developer should globally install this package on their machine rather than this project. That way installing the package would affect others.